### PR TITLE
Message structure re-arrangement

### PR DESCRIPTION
**Pull request by thakurajayL**
1. Optimize communication between s1ap container and Mme-app container
2. Optimize communication between s6a-app container and mme-app container.
This should help in improving performance.

### CODE REVIEW

File: include/common/msgType.h [hunk 1]
```diff
@@ -595,6 +590,64 @@ struct erab_release_command_Q_msg {
...
+    int s1ap_enb_ue_id;
+    int criticality;
+    unsigned char IMSI[BINARY_IMSI_LEN];
+    struct TAI tai;
+    struct CGI utran_cgi;
+    enum ie_RRC_est_cause rrc_cause;
+	struct STMSI s_tmsi;
+    int enb_fd;
+
+    struct MS_net_capab ms_net_capab;
```
Comment (thakurajayL) [inline include/common/msgType.h:605]: Need to get rid of redundant fields if any.

File: include/common/msgType.h [hunk 2]
```diff
@@ -46,11 +46,9 @@ typedef enum nas_int_algo
     NAS_INT_ALGORITHMS_EIA7,
 }nas_int_algo_enum;
 
-typedef enum msg_data_t
-{
-    ue_data = 0,
-    session_data
-}msg_data_t;
+typedef enum s1ap_msg_type {
```
Comment (thakurajayL) [inline include/common/msgType.h:49]: Removing this.

**Comment by yoooou** [comment]
retest this please

**Comment by thakurajayL** [comment]
retest this please

**Review by badhrinathpa — APPROVED** [review]
+1

**Pull request merged**